features/steps/s4s.py

```diff
@@ -120,15 +123,21 @@ def step_impl(context, mu_ccds_query):
         'Missing {query}'.format(query=query)
 
 
-@then('the resource parses as valid FHIR DSTU2 content')
-def step_impl(context):
+@then('the resource parses as valid FHIR {version_name} content')
+def step_impl(context, version_name):
+
     resource = context.response.json()
 
     assert "resourceType" in resource, \
-           "Resource has no resourceType: {res}".format(res=resource)
+        "Resource has no resourceType: {res}".format(res=resource)
+
+    server_config_key = "API_SERVER_{server_version}".format(server_version=version_name)
+
+    assert server_config_key in current_app.config, \
```

Comment:
I'm pretty sure we don't want to pull from current_app here. It probably hasn't been done in a long time, but we (hopefully still) can actually run the entire test suite from the command line just by calling `behave`, and flask wont be loaded at all. If you need to read from ENV variables, I'd suggest adding them to `testsuite/config_reader.py` and having that update the context config.

Reply:
Ah, good point. I was actually running it from the command line (and adding --tags to test) before I made this change. Will update.